src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,71 @@ th {
     }
   }
 
+  &.-portal {
+    caption {
+      background-color: set-color(grey, 30);
+      font-weight: 700;
+      padding-bottom: 0.75rem;
+      padding-left: 1rem;
+      padding-top: 0.75rem;
+      text-align: left;
+    }
+
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
+
+          &:nth-child(2) {
+            text-align: right;
+          }
+        }
+      }
+    }
+
+    tbody {
+      tr {
+        &:nth-child(odd) {
+          background-color: set-color(grey, 20);
+        }
+
+        &:hover {
```

Comment:
The blue style is intended for active/selected states, not hover. The hex code for the background-color is #cedaf1. It's similar to Navy 30 but with less saturation. Let's use it for now. I will investigate if we can use Navy 30 or develop a new color that's in between Navy 30 and  #cedaf1 and then replace the value of navy 30 with that.

Reply:
Makes sense. Sass linter doesn't allow using Hexadecimal colors, so I applied converted  rgb(206, 218, 241);. Thanks!